Core/Systems/EcsSystems.cs

```diff
@@ -9,13 +9,21 @@ public sealed partial class EcsSystems
 		private readonly EcsWorld _world;
 		private readonly List<IEcsSystem> _allSystems = new();
 
+#if SECS_PROFILING
+		private readonly EcsProfiler _ecsProfiler;
```

Comment:
Compilation error due to missing namespace (don't forget to put it inside preprocessor if)